src/features/search/components/SearchAutocompleteItem/SearchAutocompleteItem.tsx

```diff
@@ -5,27 +5,39 @@ import { Keyboard, Text } from 'react-native'
 import styled from 'styled-components/native'
 import { v4 as uuidv4 } from 'uuid'
 
+import { SearchGroupNameEnumv2 } from 'api/gen'
 import { UseNavigationType } from 'features/navigation/RootNavigator/types'
 import { getTabNavConfig } from 'features/navigation/TabBar/helpers'
 import { Highlight } from 'features/search/components/Highlight/Highlight'
 import { useSearch } from 'features/search/context/SearchWrapper'
-import { useAppliedFilters } from 'features/search/helpers/useAppliedFilters/useAppliedFilters'
+import {
+  FILTER_TYPES,
+  useAppliedFilters,
+} from 'features/search/helpers/useAppliedFilters/useAppliedFilters'
 import { SearchView } from 'features/search/types'
 import { AlgoliaSuggestionHit } from 'libs/algolia'
+import { env } from 'libs/environment'
 import { analytics } from 'libs/firebase/analytics'
+import { useSearchGroupLabel } from 'libs/subcategories'
 import { MagnifyingGlass } from 'ui/svg/icons/MagnifyingGlass'
-import { getSpacing } from 'ui/theme'
+import { getSpacing, Typo } from 'ui/theme'
 
 type Props = {
   hit: AlgoliaSuggestionHit
   sendEvent: SendEventForHits
+  shouldShowCategory: boolean
```

Comment:
```suggestion
  shouldShowCategory?: boolean
```
j'aime bien qu'un maximum de props soient optionnelles. Surtout lorsqu'il s'agit d'un `boolean` où l'on peut considérer que `false` et `undefined` produisent le même résultat.

Ca évite d'être obligé de faire `< SearchAutocompleteItem showShowCategory={false} />` à chaque fois